### ISSUE TITLE
Ash fall config option

### ISSUE THREAD

**Issue by T0paz** [TerraformersMC/Cinderscapes#42]
What it says on the tin. Whenever I enter the Ashy Shoals biome, my framerates drop significantly, seemingly due to all the snowfall in the region. The ash sheets seem to place in the world a lot faster than snow, but I may be wrong. I think the ideal solution to this issue would be to add configuration which lets you control the ash fall, or remove it entirely for weaker PCs (though, my PC is by no means weak).

**Comment by ghost** [comment]
Yep, we’re definitely going to be adding a config for that in the final release. Just curious do you think it’s the actual block placement or the particles that’s doing it?

**Comment by T0paz** [comment]
I just built the mod from source to test how it'd run both with and without snowfall in Ashy Shoals, but when I went in game, it looks like the snowfall was already disabled. But, it also resulted in my framerates not dipping, so I'm pretty sure the (ash) snow layers being added all over the magma blocks at a quick pace was causing the lag. 

EDIT: I was in spectator, which prevents snowfall from happening around you. Enabling snow fall actually *didn't* drop my frames, so I'll have to look into possible mod incompatibilities.
